[FLINK-15792][k8s] Make Flink logs accessible via kubectl logs per default (#12240)
apache/flink: docs/ops/deployment/native_kubernetes.md

## Log Files

> By default, the JobManager and TaskManager will output the logs to console and `/opt/flink/log` in each pod simultaneously.

Reviewer: ```suggestion
By default, the JobManager and TaskManager will output the logs to the console and `/opt/flink/log` in each pod simultaneously.
```

> The STDOUT and STDERR will only be redirected to console. You could check for them via `kubectl logs <PodName>`.

Reviewer: ```suggestion
The STDOUT and STDERR will only be redirected to the console. You can access them via `kubectl logs <PodName>`.
```
